R/HLBoot.r

```diff
@@ -18,7 +18,7 @@
 #' @param bca Boolean whether to estimate BCa intervals for MF.
 #' @param return.boot Boolean whether to save the bootstrap samples of the statistics.
 #' @param trace.it Boolean whether to display verbose tracking of the cycles.
-#' @param seed initial seed value. Ignored.
+#' @param seed Used for set.seed.
```

Comment:
Description should be informative regarding purpose. Why would a user care about set.seed?

```suggestion
#' @param seed Used to initialize random number generator for reproducibility. Passed to \code{set.seed}.
```